src/server/test/hotspot_partition_test.cpp

```diff
@@ -92,6 +121,37 @@ TEST_F(hotspot_partition_test, hotspot_partition_policy)
     test_rows[HOT_SCENARIO_1_WRITE_HOT_PARTITION].put_qps = 5000.0;
     expect_vector = {{0, 0, 0, 4, 0, 0, 0, 0}, {0, 0, 4, 0, 0, 0, 0, 0}};
     test_policy_in_scenarios(test_rows, expect_vector, calculator);
+    clear_calculator_histories(calculator);
+}
+
+TEST_F(hotspot_partition_test, send_hotkey_detect_request)
+{
+    const int READ_HOT_PARTITION = 7;
+    const int WRITE_HOT_PARTITION = 0;
+    std::vector<row_data> test_rows = generate_row_data();
+    test_rows[READ_HOT_PARTITION].get_qps = 5000.0;
+    test_rows[WRITE_HOT_PARTITION].put_qps = 5000.0;
+    int hotpartition_count = FLAGS_occurrence_threshold;
+    std::vector<std::array<int, 2>> expect_result = {{0, hotpartition_count},
+                                                     {0, 0},
+                                                     {0, 0},
+                                                     {0, 0},
+                                                     {0, 0},
+                                                     {0, 0},
+                                                     {0, 0},
+                                                     {hotpartition_count, 0}};
+    aggregate_analyse_data(calculator, test_rows, expect_result, FLAGS_occurrence_threshold);
+    const int back_to_normal = 30;
+    hotpartition_count = FLAGS_occurrence_threshold - back_to_normal;
+    expect_result = {{0, hotpartition_count},
+                     {0, 0},
+                     {0, 0},
+                     {0, 0},
+                     {0, 0},
+                     {0, 0},
+                     {0, 0},
+                     {hotpartition_count, 0}};
```

Comment:
`expect_result` is set on Line 142, why set the same value again?

Reply:
It is a different test case, expext result changed